interactions.go

```diff
@@ -90,11 +95,16 @@ type Interaction struct {
 	Version int    `json:"version"`
 }
 
-// ApplicationCommandInteractionData contains data received in an interaction event.
-type ApplicationCommandInteractionData struct {
+// InteractionData contains data received from InteractionCreate event.
+type InteractionData struct {
```

Comment:
Dont change the name please. It is a breaking change to rename this

Reply:
Alright, let's keep `ApplicationCommandInteractionData ` if the data isn't for not only for application commands. Besides, there was no release of Slash Commands... so this is not a breaking change. Those who wanted master branch were aware, that it may contain breaking changes.

Reply:
> Besides, there was no release of Slash Commands... so this is not a breaking change. Those who wanted master branch were aware, that it may contain breaking changes.

Yup thats fair enough :+1: